.gitignore

```diff
@@ -4,6 +4,7 @@ out/
 !**/src/test/**/out/
 
 ### Maven ###
+test/
```

Comment:
Why are you ignoring this?

Reply:
Usually test folder is kept in GitHub, for what I know

Reply:
Should've ignored the resources folder, forgot to change it